src/client/content/styles_delayed.css

```diff
@@ -3,25 +3,34 @@
 .sbs-web .ui-overlay-a, 
 .sbs-web .ui-page-theme-a, 
 .sbs-web .ui-page-theme-a .ui-panel-wrapper {
-    background-image: url('v1/images/main_image_mobile2.jpg');
+ /* background-image: url('v1/images/main_image2.jpg');*/
+        background-image:none;
+        background-color: #32394c;
 }
 
 .sbs-cdb .ui-overlay-a, 
 .sbs-cdb .ui-page-theme-a, 
 .sbs-cdb .ui-page-theme-a .ui-panel-wrapper {
-    background-image: url('../images/main_image_mobile2.jpg');
+ /* background-image: url('v1/images/main_image2.jpg');*/
```

Comment:
Don't put images under v1 directory for styles prefixed with .sbs-cdb. Instead they should be under ../images/, just like existing images